### PR TITLE
grass.jupyter: Ensure width and height in InteractiveMap are integers 

### PR DESCRIPTION
**Pull request by cwhite911**
The users input wasn't getting checked here causing the query to fail if the user defined the `width` parameter as a string.

The code below would render the map and fail when the query_vector tried to return.

 ```
m = gj.InteractiveMap(width="500", map_backend="ipyleaflet")
```

### CODE REVIEW

**Comment by cwhite911** [comment]
> This should be solved in InteractiveMap constructor. Simplest would be to do:
> 
> ```
>         self.width = int(width)
>         self.height = int(height)
> ```
> 
> and let it throw the exception.

That makes sense.

File: python/grass/jupyter/interactivemap.py [hunk 1]
```diff
@@ -764,8 +764,8 @@ def __init__(
         self._ipywidgets = ipywidgets
         self.raster_name = rasters
         self.vector_name = vectors
-        self.width = width
         self.query_control = None
+        self.width = width
```
Comment (petrasovaa) [inline python/grass/jupyter/interactivemap.py:768]: This looks like a leftover change that is not needed.

Comment (cwhite911) [reply python/grass/jupyter/interactivemap.py:768]: I doubled checked and it looks like this is needed. The width is getting pass in by `InteractiveMap._create_toggle_button.` when building the `InteractiveQueryController`.

Comment (petrasovaa) [reply python/grass/jupyter/interactivemap.py:768]: Right, but it's already there, you are just moving the line.

**Pull request merged**